out/com/chapter2/NutritionFacts.java

```diff
@@ -0,0 +1,62 @@
+package com.chapter2;
+
+public class NutritionFacts {
```

Comment:
please add some javadoc describng the purpose of the class.

Reply:
okay, i will